modules/learn/pages/security/roles.adoc

```diff
@@ -1943,6 +1943,59 @@ It allows the reading of bucket settings.
 ^| image:introduction/no.png[]
 |===
 
+
+[#query-sequential-scan]
+== Query Use Sequential Scan
+
+The *Query Use Sequential Scan* role, located under Query & Index in the Web Console's roles list, allows users' queries to perform a sequential scan of a keyspace.
+The query planner only decides to use a sequential scan when there is no suitable index for the keyspace. 
+Only queries by users with this role can use a sequential scan to access data because scanning a large unindexed keyspace can be expensive.  
+This role does not grant the user the ability to read or mutate data or access to the Web Console. 
+Administrators' queries automatically have permission to perform sequential scans when necessary.
+
+[#table_query_delete_role,cols="15,8,8,8,8",hrows=3]
+|===
+5+^| Role: Query Use Sequential Scan (`query_use_sequential_scans`)
+
+.2+^h| Resources
+4+^h| Privileges
+
+^h| *Read*
+^h| *Write*
+^h| *Execute*
+^h| *Manage*
+
+^| query_use_seqential_scans
```

Comment:
Got to confess, I don't fully understand the layout of the tables on this page, but I think it looks inconsistent having the underlying role name listed here. The other tables use natural language descriptions in the Resources column. Maybe replace this text with "Sequential Scans" or "Query Sequential Scans"? The role name is given at the top of the table.

Reply:
Makes sense. I've made the change and updated the perview site.